### PR TITLE
Add new Ratio Check QC test.

### PR DESCRIPTION
**Pull request by cameronbook**
Added QC Ratio Check.

Screen Shot 2020-12-10 at 4 44 54 PM
Screen Shot 2020-12-10 at 4 21 47 PM

### CODE REVIEW

**Review by emilyhcliu — DISMISSED** [review]
Looks good!

File: src/ufo/filters/QCflags.h [hunk 1]
```diff
@@ -27,7 +27,11 @@ namespace QCflags {
   constexpr int buddy   = 13;  // observation rejected by the buddy check
   constexpr int derivative = 14;  // observation removed due to metadata derivative value
   constexpr int profile = 15;  // observation rejected by at least one profile QC check
+<<<<<<< HEAD
```
Comment (CoryMartin-NOAA) [inline src/ufo/filters/QCflags.h:30]: This wasn't fixed when you merged in develop, also note that there are two number 16s!

File: src/ufo/filters/QCmanager.cc [hunk 2]
```diff
@@ -171,13 +174,12 @@ void QCmanager::print(std::ostream & os) const {
       if (iseaice  > 0) os << info << iseaice  << " removed by sea ice check." << std::endl;
       if (itrack   > 0) os << info << itrack  << " removed by track check." << std::endl;
       if (ibuddy   > 0) os << info << ibuddy  << " removed by buddy check." << std::endl;
+      if (iratioref > 0) os << info << iratioref << " rejected by ratio check." << std::endl;
       if (ionedvar  > 0) os << info << ionedvar  << " removed by 1D Var check." << std::endl;
-
-      os << info << ipass << " passed out of " << iobs << " observations." << std::endl;
```
Comment (CoryMartin-NOAA) [inline src/ufo/filters/QCmanager.cc:176]: This line shouldn't be deleted.

File: src/ufo/filters/QCmanager.cc [hunk 3]
```diff
@@ -171,13 +174,12 @@ void QCmanager::print(std::ostream & os) const {
...
     }
 
     ASSERT(ipass + imiss + ipreq + ibnds + iwhit + iblck + iherr + ithin + iclw + iprof + ifgss + \
-           ignss + idiffref + iseaice + itrack + ibuddy + idydx  + ionedvar == iobs);
```
Comment (CoryMartin-NOAA) [inline src/ufo/filters/QCmanager.cc:180]: This wasn't merged properly

**Review by CoryMartin-NOAA — CHANGES_REQUESTED** [review]

**Pull request closed without merging**